templates/base.html

```diff
@@ -44,28 +44,27 @@
                         <!-- Nav -->
                         <div class="show-for-small-only">
                             <ul class="nav">
-                                <li><button type="button" class="menu-icon" data-toggle="offCanvas"></button></li>
+                                <li class="small-menu"><button type="button" class="menu-icon" data-toggle="offCanvas"></button></li>
                             </ul>
                         </div>
                         <div class="hide-for-small-only">
                             <ul class="nav">
                                 <li><a class="active" href="{% url 'home' %}">HOME</a></li>
                                 <li><a class="active" href="{% url 'artists' %}">ARTISTS</a></li>
                                 {% if user.is_authenticated %}
-                                    <li style="float: right;"><a class="active" href="{% url 'logout' %}">LOGOUT</a></li>
-                                    <li style="float: right;"><a class="active" href="{% url 'profile' %}">PROFILE</a></li>
+                                    <li style="float: right;"><a href="{% url 'logout' %}">LOGOUT</a></li>
+                                    <li style="float: right;"><a href="{% url 'profile' %}">PROFILE</a></li>
                                 {% else %}
                                     <li style="float: right;"><a data-open="login-modal">LOGIN</a></li>
                                 {% endif %}
                             </ul>
                         </div>
-                    </div>
                     <!-- Content -->
                     <div class="container">
                         {% block content %}
                         {% endblock %}
                     </div>
-
+                </div>
```

Comment:
By removing the `<div>` above and adding a new one here, it looks like you're trying to add the Content section to the `"full-page"` div. If that's the case, you should indent this `</div>` in 4 spaces here and indent the entire Content section.

So basically what you have now is:
```html
                ...
                {% endif %}
            </ul>
        </div>
    <!-- Content -->
    <div class="container">
        {% block content %}
        {% endblock %}
    </div>
</div>
...
```

It should look like:
```html
                ...
                {% endif %}
            </ul>
        </div>
        <!-- Content -->
        <div class="container">
            {% block content %}
            {% endblock %}
        </div>
    </div>
    ...
```